aiida_optimize/engines/_base.py

```diff
@@ -58,6 +58,13 @@ def is_finished(self) -> bool:
         Returns true if the optimization is finished.
         """
 
+    @property
+    def is_finished_ok(self) -> bool:
+        """
+        Returns true if the optimization is finished without error.
+        """
+        True
```

Comment:
Ha, missed that one before checking CI: should be `return True`.

Reply:
Changed as noted above so that it returns `self.is_finished`